insight-rest-client/src/main/java/org/nuxeo/ai/sdk/rest/Common.java

```diff
@@ -20,6 +20,9 @@
 
 package org.nuxeo.ai.sdk.rest;
 
+/**
+ * Common Constants
+ */
 public final class Common {
```

Comment:
I like to put the name Constants in the Class name when all it does is keep constants.

Reply:
it's a possible placeholder for common functions as well.